core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/store/content-category-field.store.ts

```diff
@@ -285,7 +292,7 @@ export const CategoryFieldStore = signalStore(
                                 },
                                 error: () => {
                                     // TODO: Add Error Handler
```

Comment:
Remove

Reply:
We need to add the dotErrorServices of Dotcms.